exercises/F-predicates/exercise.js

```diff
@@ -3,13 +3,47 @@
   ---------------------------------
   Write two predicate functions
   The variables should have values that match the expected results.
-*/
+  */
+
+  function checkAge(age){
+  if(age >= 18){
+  console.log("You are qualified");
+   }
+  else if (age < 18 && age > 1) {
+    console.log("You are a child");
+  }
+   else{
+     console.log("No age");
+   }
+ }
+console.log(checkAge(21));
+console.log(checkAge(12));
+console.log(checkAge(0));
+
+function isOdd(num) {
+  return num % 2 === 1;
+}
+console.log(isOdd(3));
+console.log(isOdd(2));
+
 
 // Finish the predicate function to test if the passed number is negative (less than zero)
-function isNegative(number) {}
+function isNegative(number) {
+  if (number < 0) {
+    return true;
+  } else {
+    return false;
+  }
+}
 
 // Finish the predicate function to test if the passed number is between 0 and 10
-function isBetweenZeroAnd10(number) {}
+function isBetweenZeroAnd10(number) {
+  if (number >= 0 && number < 11) {
```

Comment:
> Great use of if statement. However, the condition check here includes the numbers 0 through to 10 but the function requires you to return the numbers between 0 and 10. Personally, I would make a small adjustment to this condition to make it precise so that it only returns numbers 1,2,3,4,5,6,7,8 or 9.

Yeah, the question was a bit tricky for me, so I played it safe. I have corrected it. Thank you Selchuk.